The **(Build CoreCLR Tools Unit Tests linux x64 checked)** failure is real. Otherwise looks good to merge, thanks!

```
src/coreclr/tools/aot/Mono.Linker.Tests/TestCasesRunner/ILCompilerDriver.cs(65,30): error CS7036: (NETCORE_ENGINEERING_TELEMETRY=Build) There is no argument given that corresponds to the required parameter 'generateLibraryAndModuleInitializers' of 'MainMethodRootProvider.MainMethodRootProvider(EcmaModule, IReadOnlyCollection<MethodDesc>, bool)'
```

--- a/src/coreclr/tools/aot/Mono.Linker.Tests/TestCasesRunner/ILCompilerDriver.cs
+++ b/src/coreclr/tools/aot/Mono.Linker.Tests/TestCasesRunner/ILCompilerDriver.cs
@@ -62,7 +62,7 @@ namespace Mono.Linker.Tests.TestCasesRunner
 				compilationRoots.Add (new ExportedMethodsRootProvider (module));
 			}
 
-			compilationRoots.Add (new MainMethodRootProvider (entrypointModule, CreateInitializerList (typeSystemContext, options)));
+			compilationRoots.Add (new MainMethodRootProvider (entrypointModule, CreateInitializerList (typeSystemContext, options), generateLibraryAndModuleInitializers: true));
 
 			ILProvider ilProvider = new NativeAotILProvider ();
 

